pkg/operators/operators.go

```diff
@@ -100,7 +101,7 @@ type ContainerInfoFromNetNSID interface {
 
 type ContainerInfoSetters interface {
 	NodeSetter
-	SetContainerInfo(pod, namespace, container string)
+	SetContainerMetadata(*types.BasicK8sMetadata, *types.BasicRuntimeMetadata, bool)
```

Comment:
I find it weird to have a `setContainerInfo` bool param in a `SetContainerMetadata` function. What about splitting this into `SetContainerMetadata` (as setContainerInfo=true) and `SetPodMetadata` (as setContainerInfo=false) and removing the boolean?

Reply:
Yep, it was also strange for me. Thanks for the suggestion. Applied.